src/linter-service/server.ts

```diff
@@ -150,7 +150,8 @@ async function startServerInternal(webContainer: WebContainer): Promise<ServerIn
 	const serverInternal = {
 		process: serverProcess,
 		request,
-		ready: request('ok?', (res) => res === 'ok' || res === 'boot').then(() => {
+		ready: request('ok?', (res) => res === 'ok' || res === 'boot').then(async () => {
+			await new Promise((resolve) => setTimeout(resolve, 100));
```

Comment:
When I debugged locally, there was an issue where the server would stop if I requested lint immediately after starting the server. I changed it to wait a little bit for the server to stabilize.